src/main/java/org/broadinstitute/hellbender/tools/spark/sv/ISZMetricsCollectorSpark.java

```diff
@@ -0,0 +1,150 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import htsjdk.samtools.SAMReadGroupRecord;
+import htsjdk.samtools.SamPairUtil;
+import htsjdk.samtools.metrics.MetricsFile;
+import org.apache.spark.api.java.JavaRDD;
+import org.broadinstitute.hellbender.metrics.MetricAccumulationLevel;
+import org.broadinstitute.hellbender.tools.picard.analysis.InsertSizeMetrics;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeMap;
+
+/**
+ * Worker class to collect insert size metrics.
+ */
+public class ISZMetricsCollectorSpark implements Serializable{
```

Comment:
rename to clarify the purpose - `ISZMetricsCollector` is totally unreadable. Is it a `InsertSizeMetricsCollector`?